Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

```diff
@@ -232,6 +232,9 @@ stretched) then you will have 2 replicas per datacenter where each replica ends
         * `disabled`: whether to enable or disable pool mirroring status
         * `interval`: time interval to refresh the mirroring status (default 60s)
 
+    !!! note
```

Comment:
Instead of adding this note here, let's copy the `statusCheck` section to the rados namespace CRD doc. Otherwise, a user looking at the rados namespace doc won't see anything about the status.
```suggestion
```

Reply:
@travisn we didn't have a `statusCheck` spec in the RadosNamespace CRD, we are just using the the values from `cephBlockPool` CRD,

Is your recommendation to add a `statusCheck` in the radosNamespace CRD, and also allow user to adjust the values if needed?

Reply:
Ok, now I see that the statusCheck settings are just used from the blockpool. IMO the comment just needs to be in the rados namespace doc since that's where users will be looking for rados namespace mirroring settings. How about just moving this note into ceph-block-pool-rados-namespace-crd.md, around line 59? And rephrase it something like this?
```suggestion
    !!! note
    If mirroring is enabled, whether to monitor the status and the interval of status updates is based on the `statusCheck` spec values of the parent CephBlockPool CR.

```